proposals/dao/vip_13.ts

```diff
@@ -0,0 +1,174 @@
+  await voltMigrator.deployed();
+
+  console.log(`\nVolt Migrator deployed to: ${voltMigrator.address}`);
+
+  const MigratorRouter = await ethers.getContractFactory('MigratorRouter');
+  const migratorRouter = await MigratorRouter.deploy(
+    addresses.core,
+    voltV2.address,
+    daiPriceBoundPSM.address,
+    usdcPriceBoundPSM.address
+  );
+  await migratorRouter.deployed();
+
+  console.log(`\nMigrator Router deployed to: ${migratorRouter.address}`);
+
+  return {
+    // put returned contract objects here
+  };
+};
+
+// Do any setup necessary for running the test.
+// This could include setting up Hardhat to impersonate accounts,
+// ensuring contracts have a specific state, etc.
+const setup: SetupUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {
+  console.log(`No actions to complete in setup for vip${vipNumber}`);
+};
+
+// Tears down any changes made in setup() that need to be
+// cleaned up before doing any validation checks.
+const teardown: TeardownUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {
+  console.log(`No actions to complete in teardown for vip${vipNumber}`);
+};
+
+// Run any validations required on the vip using mocha or console logging
+// IE check balances, check state of contracts, etc.
+const validate: ValidateUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {
+  const {
+    erc20Allocator,
+    PCVGuardian,
+    voltV2UsdcPriceBoundPSM,
+    voltV2DaiPriceBoundPSM,
+    daiCompoundPCVDeposit,
+    usdcCompoundPCVDeposit
+  } = contracts;
+
+  expect(PCVGuardian.isWhitelistAddress(voltV2UsdcPriceBoundPSM.address)).to.be.true;
+  expect(PCVGuardian.isWhitelistAddress(voltV2DaiPriceBoundPSM.address)).to.be.true;
+
+  {
+    const [token, targetBalance, decimalsNormalizer] = await erc20Allocator.allPSMs(voltV2DaiPriceBoundPSM.address);
+    const psmAddress = await erc20Allocator.pcvDepositToPSM(daiCompoundPCVDeposit.address);
+    expect(psmAddress).to.be.equal(voltV2DaiPriceBoundPSM.address);
+    expect(token).to.be.equal(addresses.dai);
+    expect(daiTargetBalance.toString()).to.be.equal(targetBalance.toString()); /// had to make it a string otherwise typescript threw an error about comparing objects
+    expect(decimalsNormalizer.toString()).to.be.equal(daiDecimalsNormalizer.toString());
+  }
+
+  {
+    const [token, targetBalance, decimalsNormalizer] = await erc20Allocator.allPSMs(voltV2UsdcPriceBoundPSM.address);
+    const psmAddress = await erc20Allocator.pcvDepositToPSM(usdcCompoundPCVDeposit.address);
+    expect(psmAddress).to.be.equal(voltV2UsdcPriceBoundPSM.address);
+    expect(token).to.be.equal(addresses.usdc);
+    expect(usdcTargetBalance.toString()).to.be.equal(targetBalance.toString()); /// had to make it a string otherwise typescript threw an error about comparing objects
+    expect(decimalsNormalizer.toString()).to.be.equal(usdcDecimalsNormalizer.toString());
+  }
```

Comment:
Add assertions that Volt token, migrator and router contracts are deployed correctly